packages/test/test-end-to-end-tests/src/test/gc/gcAttachmentBlobs.spec.ts

```diff
@@ -0,0 +1,280 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { stringToBuffer } from "@fluidframework/common-utils";
+import { IContainer } from "@fluidframework/container-definitions";
+import { Container } from "@fluidframework/container-loader";
+import { ContainerRuntime } from "@fluidframework/container-runtime";
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { IOdspResolvedUrl } from "@fluidframework/odsp-driver-definitions";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+import { ITestContainerConfig, ITestObjectProvider } from "@fluidframework/test-utils";
+import { describeNoCompat, ITestDataObject } from "@fluidframework/test-version-utils";
+import { getUrlFromItemId, MockDetachedBlobStorage } from "../mockDetachedBlobStorage";
+import { getGCStateFromSummary } from "./gcTestSummaryUtils";
+
+const ensureContainerConnectedWriteMode = async (container: Container) => {
+    const resolveIfActive = (res: () => void) => { if (container.deltaManager.active) { res(); } };
+    if (!container.deltaManager.active) {
+        await new Promise<void>((resolve) => container.on("connected", () => resolveIfActive(resolve)));
+        container.off("connected", resolveIfActive);
+    }
+};
+
+/**
+ * Validates that unreferenced blobs are marked as unreferenced and deleted correctly.
+ */
+describeNoCompat("Garbage collection of blobs", (getTestObjectProvider) => {
```

Comment:
Is this just a complete test move? I scanned this and didn't see any differences of note

Reply:
Yes, it's just a move.